docs/tutorials/grover_with_sampler.ipynb

```diff
@@ -0,0 +1,523 @@
+      "text/plain": [
+       "<Figure size 538.279x264.88 with 1 Axes>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# With 2 iterations\n",
+    "grover_circuits[1].draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed5a7c5f-5f21-40d4-b4d1-37f8bb46337e",
+   "metadata": {},
+   "source": [
+    "## 4. Submit the circuits to IBM Cloud\n",
+    "\n",
+    "Now that the Grover's circuits are created, let's submit them to a quantum computer on IBM Cloud by using the Sampler program."
```

Comment:
Gentle reminder to make this generic as well, for IBM Cloud or IBM Quantum when fixing Step 1.

Reply:
I change 224 to "Submit the circuits to a quantum computer on the cloud" and 226 to "a quantum computer on the cloud". What do you think?